Makefile

```diff
@@ -148,7 +148,7 @@ src/prebuilt/wast-parser-gen.cc: src/wast-parser.y
 	bison -o $@ $< --defines=src/prebuilt/wast-parser-gen.hh --report=state
 
 src/prebuilt/wast-lexer-gen.cc: src/wast-lexer.cc
-	re2c --no-generation-date -bc -o $@ $<
+	re2c --no-generation-date -bc8 -o $@ $<
```

Comment:
The Makefile generates the prebuilt lexer/parser. The `CMakeLists.txt` one is used if re2c/bison are on your machine and new enough. It'd be nice to remove the prebuilt, but I don't know if I wanna require those tools for windows, since it's kind of a pain.